addons/hr_holidays/models/hr_leave.py

```diff
@@ -1136,6 +1136,18 @@ def action_approve(self):
     def _get_leaves_on_public_holiday(self):
         return self.filtered(lambda l: l.employee_id and not l.number_of_days)
 
+    def _get_employee_with_holiday_type(self):
```

Comment:
As we return multiple employees, let's make the method name reflect that
```suggestion
    def _get_employees_from_holiday_type(self):
```

Reply:
Thank you so much for looking into it, I hope it will be merged soon !!